docs/getting_started/dashboard_copilot.zh.md

```
@@ -157,127 +152,409 @@ agent:
-    你可以使用 `datus-agent service add` 交互式添加 SQL 数据源；semantic layer 和 BI platform 配置仍需要在 YAML 中编辑。
+    也可以在 REPL 内通过斜杠命令交互式添加：`/datasource` 添加 SQL 数据源，`/services` 添加 semantic layer、BI platform 与 scheduler。
 
 ## 步骤 3：从仪表盘 Bootstrap
 
-现在使用 `bootstrap-bi` 命令从 Superset 仪表盘自动生成上下文和子代理。我们将以世界银行数据仪表盘为例。
+在 Datus REPL 内使用 `/bootstrap-bi` 斜杠命令，从 Superset 仪表盘自动生成上下文和子代理。我们将以世界银行数据仪表盘为例。
 
-### 运行 Bootstrap 命令
+### 启动 REPL
 
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)